lib/js/atoms/Badge/Badge.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="badge" :class="{ '-minor': color === BADGE_COLORS.MINOR }">
+		<div class="badge__icon">
+			<slot />
+		</div>
+		<div class="badge__label">{{ label }}</div>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/atoms/icons';
+
+.badge {
```

Comment:
It conflicts with https://github.com/bethinkpl/wnl-platform/blob/1b5982c53778a140028e05135f157f523fd41696/resources/assets/js/components/global/Avatar.vue#L46-L51 and https://github.com/bethinkpl/wnl-platform/blob/1b5982c53778a140028e05135f157f523fd41696/resources/assets/styles/_mixins.scss#L77-L90.

Reply:
but this classes are scoped

Reply:
btw this is public repo